lib/util/stats.c

```diff
@@ -107,7 +108,7 @@ int stats_print(struct stats_record *stats_rec, struct stats_record *stats_prev)
 			return 0;
 
 		if (first) {
-			printf("Period of %fs ending at %lu.%06lu\n", period,
+			printf("Period of %fs ending at %" PRId64 ".%06lu\n", period,
```

Comment:
Let's be consistent and replace the second `lu` here as well...

Reply:
Does not work:
```
stats.c: In function 'stats_print':
stats.c:111:32: error: format '%llu' expects argument of type 'long long unsigned int', but argument 4 has type 'long int' [-Werror=format=]
  111 |                         printf("Period of %fs ending at %" PRIu64 ".%06" PRIu64 "\n", period,
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                                t.tv_sec, t.tv_nsec / 1000);
      |                                          ~~~~~~~~~~~~~~~~
      |                                                    |
      |                                                    long int
In file included from stats.c:9:
```

Reply:
I restructured now the PR:
https://github.com/xdp-project/xdp-tools/pull/206/commits/6ebbbb221b8edf19a4b6262a51fa83710a4e47c8